sdk/tables/ci.yml

```diff
@@ -0,0 +1,42 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+
+trigger:
+  branches:
+    include:
+      - master
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/tables/
+
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/tables/
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+    parameters:
+      ServiceDirectory: tables
+      Artifacts:
```

Comment:
if you add an artifact for this package, I think the automatic pipelines will want to publish a nightly dev build for you, which maybe you don't want if you're going to change the package name soon